packages/next/pages/_document.tsx

```diff
@@ -978,7 +978,11 @@ export class NextScript extends Component<OriginProps> {
 
       if (largePageDataBytes && bytes > largePageDataBytes) {
         console.warn(
-          `Warning: data for page "${__NEXT_DATA__.page}" is ${prettyBytes(
+          `Warning: data for page "${__NEXT_DATA__.page}"${
+            __NEXT_DATA__.page === context.dangerousAsPath
```

Comment:
What's the reason this is called `dangerousAsPath`?

Reply:
I think we named it this to stress it wasn't normalized/sanitized although it was mainly added for rendering the `canonical` meta tag for AMP.